shipshape.yml

```diff
@@ -187,13 +187,7 @@ checks:
     - name: '[FILE] Yaml lint theme files'
       severity: high
       path: web/themes
```

Comment:
Same here, the 'themes` path is only relevant on SaaS running in containers so perhaps needs to be overriden in CI?